### PR TITLE
Add AzureCLI 2.37.0

### PR DESCRIPTION
**Pull request by azclibot**
Add AzureCLI 2.37.0

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/61802)

### CODE REVIEW

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Need to move to multi-file manifest

File: manifests/m/Microsoft/AzureCLI/2.37.0/Microsoft.AzureCLI.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+PackageIdentifier: Microsoft.AzureCLI
+PackageVersion: 2.37.0
+PackageName: Microsoft Azure CLI
+Publisher: Microsoft Corporation
+Moniker: azure-cli
+PackageUrl: https://docs.microsoft.com/en-us/cli/azure/
+License: Copyright (C) Microsoft Corporation
```
Comment (Trenly) [inline manifests/m/Microsoft/AzureCLI/2.37.0/Microsoft.AzureCLI.yaml:7]: A copyright is also not a license

**Comment by Trenly** [comment]
It seems the Azure CLI bot always uses singletons, incorrect installer type, Copyright instead of license, and msi instead of wix installer type.

@kairu-ms - Is this something the azure CLI team might be able to fix up quickly, or is there a different team which handles the bot?

**Comment by jedieaston** [comment]
Some people who wrote their own integrations for CI claimed that it was harder to manually create all of the files vs just swapping some fields in a singleton (where they can just have a template). They should probably be using wingetcreate, but I doubt Azure CLI is built on Windows.

**Pull request closed without merging**